webrtc.html

```diff
@@ -2604,6 +2606,40 @@
             </dl>
           </section>
         </div>
+        </section>
+      <section>
+        <h4>Configuration data extensions</h4>
+        <div>
+          <pre class="idl">partial dictionary RTCOfferOptions {
+            boolean offerToReceiveAudio;
+            boolean offerToReceiveVideo;
+          };
+          </pre>
+          <section>
+            <h2>Attributes</h2>
+            <dl data-link-for="RTCOfferOptions" data-dfn-for="RTCOfferOptions" class="dictionary-members">
+              <dt><dfn>offerToReceiveAudio</dfn> of type <span class="idlMemberType"><a>boolean</a></span></dt>
+              <dd>
+                <p>
+                  When this is given a non-false value, and no outgoing track of
+                  type "audio" is attached to the PeerConnection, createOffer()
+                  will behave as if addTransceiver("audio") had been called prior
```

Comment:
I also added a precondition that no existing recv-able media section exists. Otherwise, doing a re-offer with offerToReceive=true would add an extra media section for each re-offer, and I don't think that was the intended behavior.